lib/jsdom/browser/xhr/formdata.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+var stream = require("stream");
+
+var Blob = require("../file/blob");
+
```

Comment:
Since this is an incomplete implementation of FormData, add a TODO note at the top to finish it out.